src/ansys/geometry/core/modeler.py

```diff
@@ -390,8 +390,8 @@ def open_file(
         """Open a file.
 
         This method imports a design into the service. On Windows, ``.scdocx``
-        and HOOPS Exchange formats are supported. On Linux, only the ``.scdocx``
-        format is supported.
+        and reader formats are supported. On Linux, only the ``.scdocx``
+        and reader formats are supported.
 
```

Comment:
I think this needs to be rewritten now.. both the Linux and Windows sentences say the same thing now (if I am not mistaken)